ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+<!--- Remove text and sections that do not apply -->
+
+This issue tracker is only for technical issues related to zcashd.
+
+General zcash questions and/or support requests and are best directed to the [Zcash Forums](https://forum.z.cash) or [Community Rocket.Chat](https://chat.zcashcommunity.com).
+
+For reporting security vulnerabilities or for sensitive discussions with our security team, please contact [security@z.cash](mailto:security@z.cash). You can use the [GPG key](https://z.cash/gpg-pubkeys/security.asc) (fingerprint: `AF85 0445 546C 18B7 86F9  2C62 88FB 8B86 D8B5 A68C`) to send an encrypted message. The key and fingerprint are duplicated on our [Public Keys page](https://z.cash/support/pubkeys.html).
+
+### Describe the issue
+
+### Can you reliably reproduce the issue?
+#### If so, please list the steps to reproduce below:
+1. 
+2. 
+3. 
+
+### Expected behaviour
+Tell us what should happen
+
+### Actual behaviour + errors
+Tell us what happens instead including any noticable error output (any messages displayed on-screen when e.g. a crash occurred)
+
+### The version of Zcash you were using:
+Run `zcashd --version` to find out
+
+### Machine specs:
+- OS name + version:
+- CPU:
+- RAM:
+- Disk size:
+- Disk Type (HD/SDD):
+- Linux kernel version (uname -a):
+- Compiler version (gcc -version):
+
+### Any extra information that might be useful in the debugging process.
+This includes the relevant contents of `~/.zcash/debug.log`. Raw text or a link to a pastebin type site are preferred.
+Please also include any non-standard things you did during compilation (extra flags, dependency version changes etc.) if applicable.
+
+### Do you have a back up of `~/.zcash directory` and/or take a VM snapshot?
```

Comment:
`of the '~/.zcash' directory` (with the correct ticks, as this comment gets rendered as Markdown).